### DISCUSSION TITLE
Remap Function ?

### DISCUSSION THREAD

**Discussion by GamesStudioTwo** [axmolengine/axmol#1904] [Q&A]
Hi there is there remap function? as I have not found it in the library I use this function a lot for box2d, here is the function I use it to set the sprite setAnchorPoint in because it maps 0 to 1. if not can you add the function to the next update of axmol its really handy.

static float MapTo(float x, float in_min, float in_max, float out_min, float out_max) {
    return (x - in_min) * (out_max - out_min) / (in_max - in_min) + out_min;
}

Example:
say you have a  slider that's is a weeerd 73px wide, but it will remap it to 0% to 100%

float myVol = MapTo( sliderPos ,0, 73, 0, 100);

Many thanks.

**Comment by halx99** [comment]
maybe you can add to ax::utils::mapTo and create a PR?

**Reply by GamesStudioTwo** [reply]
Hi thanks for quick response. is it also posable to add to the next update or axmol . as it's very handy I'm Shure lost of people would use it in the library.

**Reply by halx99** [reply]
Note: I will release the axmol-2.1.3 at  May 20, 2024

**Comment by halx99** [comment]
> Hi thanks for quick response. is it also posable to add to the next update or axmol . as it's very handy I'm Shure lost of people would use it in the library.

sure

**Reply by halx99** [reply]
And I think the remapTo chould be inline function
